docs/README.md

```diff
@@ -209,8 +209,8 @@ Where:
 - `id` - just a name, string with no additional constraints
```

Comment:
This list item is inconsistent, since it has no period at the end. Not sure how strict you want to be here, but at least I've pointed it out :)